method and results - single label function section (#22)
greenelab/text_mined_hetnet_manuscript: content/03.methods.md

For each edge type we sampled a fixed number of label functions consisting of five evenly-spaced numbers between one and the total number of possible label functions.
We repeated this sampling process 50 times for each point.
We evaluated both generative and discriminative models at each point, and we report performance of each in terms of the area under the receiver operating characteristic curve (AUROC) and the area under the precision-recall curve (AUPR).

#### Adding Random Noise to Generative Model

> Sometimes random noise can help with performance.

Reviewer: ```suggestion
We discovered in the course of this work that adding a single label function from a mismatched type would often improve the performance of the generative model (see Results).
We designed an experiment to test whether adding a noisy label function also increased performance.
```

> We discovered that adding a single label function can improve performance on top of the distant supervision model (Results Generative Model).

Reviewer: ```suggestion
```

> To better understand this anomaly we added a single label function to our distant supervision group.

Reviewer: ```suggestion
```

> This label function is designed to randomly emit either a positive or negative label at varying frequencies (five evenly spaced numbers between 0 and 1).

Reviewer: ```suggestion
This label function emitted a positive or negative label at varying frequencies, which were evenly spaced from zero to one.
Zero is the same as distant supervision alone.
```

> We trained the generative model using both distant supervision label functions and our random noise label function 

Reviewer: ```suggestion
We trained the generative model with these label functions added and report results in terms of AUROC and AUPR.
```

> Results are reported in terms of AUROC and AUPR.

Reviewer: ```suggestion
```
